translate/src/core/user/reducer.ts

```diff
@@ -69,6 +75,7 @@ export type Notifications = {
 export type UserState = {
   readonly isAuthenticated: boolean;
   readonly isAdmin: boolean;
+  readonly isNewContributor: boolean;
```

Comment:
This seems to be mixing together data that is and is not locale-dependent. Could the API and the data match instead the structure of the `managerForLocales` and `translatorForLocales` fields, with something like `contributorForLocales`? Or is there a reason why the approach proposed here is e.g. much more efficient?